service/frontend/api/handler.go

```diff
@@ -1771,6 +1771,7 @@ func (wh *WorkflowHandler) StartWorkflowExecutionAsync(
 	if err != nil {
 		return nil, fmt.Errorf("failed to encode StartWorkflowExecutionAsyncRequest: %v", err)
 	}
+	scope.RecordTimer(metrics.AsyncRequestPayloadSize, time.Duration(len(payload)))
```

Comment:
do we have the method-name dimension on this scope? e.g. `startworkflowexecution`

Reply:
yes